src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -447,7 +520,7 @@ public void loadData(NBTTagCompound nbt) {
 
     @Override
     public int onBite(IVampire biter) {
-        float perc = biter instanceof IVampirePlayer ? 1F : 0.4F;
+        float perc = biter instanceof IVampirePlayer ? 0.2F : 0.08F;
```

Comment:
This issue was present before, but it has a bigger impact with the changed percentage:

If a player has a very low blood/food bar the onBite method will calculate a value of 0 even if there is blood/food left.
So we need to ensure we suck at least 1 blood, but only if there is actually 1 blood/food left

Reply:
I think this is now resolved, as of https://github.com/TeamLapen/Vampirism/pull/355/commits/244c30b06a30c02c65cf52341bfa55f745e15cb0